Makefile

```
@@ -60,15 +60,15 @@ fmt:
 
 # Format JavaScript code
 fmt-js:
-	npx prettier --write 'static/**/*.{js,html,css}' 'test/**/*.js' '*.{json,yml,yaml,md}'
+	npx prettier --write 'internal/static/files/**/*.{js,html,css}' 'test/**/*.js' '*.{json,yml,yaml,md}'
 
 # Format all code
 fmt-all: fmt fmt-js
```

checkmake (0.3.2)

[warning] Target "fmt-all" should be declared PHONY.

(phonydeclared)